### ISSUE TITLE
Undocumented Unauthenticated Error when calling Connect-CWM

### ISSUE THREAD

**Issue by Viajaz** [christaylorcodes/ConnectWiseManageAPI#18]
I have valid ClientID provided from the ConnectWise Developer portal and I have created a public/private key set from within the ConnectWise Manage instance for my user.

I'm assuming this is a permissions issue in the ConnectWise Manage instance against `/system/info`, what are the minimum Security Role permissions required to successfully login? I notice that `/system/info` [doesn't appear to be documented under SystemSettings?](https://developer.connectwise.com/Products/Manage/REST?o=GET#/SystemSettings)

```powershell
Import-Module ConnectWiseManageAPI
$CWMConnectionInfo = @{
    Server      = "connectwise.redacted.com"
    Company     = 'RedactedCompany'
    PubKey      = 'REDACTED'
    PrivateKey  = 'REDACTED'
    ClientID    = 'REDACTED'
}
$CWM = Connect-CWM @CWMConnectionInfo -Verbose -Force
```

```
VERBOSE: GET https://connectwise.redacted.com/login/companyinfo/RedactedCompany with 0-byte payload
VERBOSE: received 289-byte response of content type application/json; charset=utf-8
VERBOSE: Using API Key authentication
VERBOSE: Validating authentication
VERBOSE: GET https://connectwise.redacted.com/v4_6_release/apis/3.0/system/info with 0-byte payload
Invoke-CWMWebRequest : An exception has been thrown.
--> ConnectWiseApi
-----> ConnectWiseApi: UserNotAuthenticated
-----> ^ Error has not been documented please report. ^
at Invoke-CWMWebRequest, C:\Program Files\WindowsPowerShell\Modules\ConnectWiseManageAPI\0.4.3.1\Private\Invoke\Invoke-CWMWebRequest.ps1: line 41
at Invoke-CWMGetMaster, C:\Program Files\WindowsPowerShell\Modules\ConnectWiseManageAPI\0.4.3.1\Private\Invoke\Invoke-CWMGetMaster.ps1: line 45  
at Get-CWMSystemInfo, C:\Program Files\WindowsPowerShell\Modules\ConnectWiseManageAPI\0.4.3.1\Public\System\Get-CWMSystemInfo.ps1: line 7        
at Connect-CWM, C:\Program Files\WindowsPowerShell\Modules\ConnectWiseManageAPI\0.4.3.1\Public\Authentication\Connect-CWM.ps1: line 163
at <ScriptBlock>, REDACTED
At C:\Program Files\WindowsPowerShell\Modules\ConnectWiseManageAPI\0.4.3.1\Private\Invoke\Invoke-CWMGetMaster.ps1:45 char:19
+ ...        $Result = Invoke-CWMWebRequest -Arguments $WebRequestArguments
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-CWMWebRequest
 
WARNING: Authentication failed. Clearing connection settings.
VERBOSE: $CWMServerConnection, variable removed.
```

**Comment by Viajaz** [comment]
Error was caused by bad Company ID, leaving issue open in case error handling of this issue was not as designed.

**Comment by christaylorcodes** [comment]
Issue resolved in version: 0.4.4.1
e459530
